driver/GalewskyShallowWaterExplicit.jl

```diff
@@ -0,0 +1,79 @@
+module GalewskyShallowWaterExplicitTests
+
+using Gridap
+using GridapGeosciences
+
+# Solves the Galewsky test case for the shallow water equations on a sphere
+# of physical radius 6371220m. Involves a shear flow instability of a zonal 
+# jet triggered by an initial gravity wave.
+# reference:
+#   Galewsky, Scott and Polvani (2004) Tellus, 56A 429-440
+
+function uθ(θϕr)
```

Comment:
I would move these three function definition statements to its own julia source file, e.g., `GalewskyInitialConditions.jl`, located at the `driver` folder. Then, we can `include(GalewskyInitialConditions.jl)` from both driver files and avoid code replication.

Reply:
done